src/connectors/accuradio.ts

```diff
@@ -20,3 +20,30 @@ function getScrollableText(selector: string) {
 		Util.getTextFromSelectors(selector)
 	);
 }
+
+function getDurationInfo() {
```

Comment:
Are you sure you cant just use Connector.timeInfoSelector = "#progressWrapper" here?

Reply:
How did I forget about this - :see_no_evil:

Reply:
I thought we had something like this!